### ISSUE TITLE
Missing authenticationPolicy in accessibility method

### ISSUE THREAD

**Issue by marekpridal** [kishikawakatsumi/KeychainAccess#413]
Based on documentation to create keychain access protected by biometry I should use this chain
```
try keychain
.accessibility(.whenPasscodeSetThisDeviceOnly, authenticationPolicy: .userPresence)
.authenticationPrompt("Authenticate to update your access token")
.set("01234567-89ab-cdef-0123-456789abcdef", key: "kishikawakatsumi")
```

but `.accessibility` method has only one parameter
```
func accessibility(_ accessibility: Accessibility) -> Keychain
```

**Comment by kishikawakatsumi** [comment]
There is another overloaded method.

```
public func accessibility(_ accessibility: Accessibility, authenticationPolicy: AuthenticationPolicy) -> Keychain {
...
```

https://github.com/kishikawakatsumi/KeychainAccess/blob/master/Lib/KeychainAccess/Keychain.swift#L463

@marekpridal

**Comment by marekpridal** [comment]
I can see this method on work computer but yesterday afternoon on my personal computer I didn't 🤦‍♂️. Anyway sorry about that, problem is on my side. Thanks for your work 🙌
